### PR TITLE
boot/zephyr: use socs conf

### PR DESCRIPTION
**Pull request by marekmatej**
Use socs folder to configure Espressif targets.
Remove all boards confs which was created by copying other boards.

### CODE REVIEW

File: boot/zephyr/socs/esp32s3_procpu.conf
Comment (almir-okato) [inline boot/zephyr/socs/esp32s3_procpu.conf:1]: Why it was an ESP32-S2 conf renamed to a ESP32-S3 one? Same configuration options?

Comment (marekmatej) [reply boot/zephyr/socs/esp32s3_procpu.conf:1]: I renamed it by mistake, but the config for the S2 and S3 is the same in this case.

**Comment by sylvioalves** [comment]
LGTM

**Review by almir-okato — APPROVED** [review]
LGTM

**Comment by sylvioalves** [comment]
@d3zd3z PTAL.

**Pull request merged**